Fabric.Identity.API/Services/CouchDbAccessService.cs

```diff
@@ -22,7 +23,12 @@ public class CouchDbAccessService : IDocumentDbService
 
         private string GetFullDocumentId<T>(string documentId)

```

Comment:
Unrelated to your changes, but in the method "Task<T> GetDocument<T>(string documentId)", there are 2 calls to "GetFullDocumentId<T>(documentId)". Perhaps we can consolidate that into a local variable instead of calling it twice.

Reply:
good call. will make that change